src/SDKs/Azure.Base/data-plane/Azure.Base/Http/Pipeline/HttpClientTransport.cs

```diff
@@ -32,10 +35,26 @@ public sealed override async Task ProcessAsync(HttpMessage message)
             var httpTransportMessage = message as Message;
             if (httpTransportMessage == null) throw new InvalidOperationException("the message is not compatible with the transport");
 
+
+            s_eventSource.ProcessingRequest(message);
             HttpRequestMessage httpRequest = httpTransportMessage.BuildRequestMessage();
 
+            var before = Stopwatch.GetTimestamp();
             HttpResponseMessage responseMessage = await ProcessCoreAsync(message.Cancellation, httpRequest).ConfigureAwait(false);
+            var after = Stopwatch.GetTimestamp();
+
             httpTransportMessage.ProcessResponseMessage(responseMessage);
+
+            if (!responseMessage.IsSuccessStatusCode) {
+                s_eventSource.ErrorResponse(message);
+            }
+
+            var elapsedMilliseconds = (after - before) * 1000 / Stopwatch.Frequency;
+            if (elapsedMilliseconds > s_delayWarningThreshold) {
+                s_eventSource.ResponseDelay(message, elapsedMilliseconds);
+            }
+
+            s_eventSource.ProcessingResponse(message);
```

Comment:
As a result of removing the logging policy, the networking code and the logging code are now all intermixed. It's not a ship stopper, but it is a small negative IMO.

Reply:
An interesting tangent from this is that we might want to log the message both when entering and exiting (being sent by transport) the pipeline.

Reply:
I agree